Glue/MobileCoinMinimal.swift

```diff
@@ -0,0 +1,34 @@
+
+
+import Foundation
+
+public enum MobileCoinMinimalError: Error {
+    case invalidReceipt
+}
+
+// MARK: -
+
+public class MobileCoinMinimal {
```

Comment:
Implements the tiny slice of the MobileCoin SDK functionality needed by the Signal iOS NSE:

* Validate incoming addresses (ensure they parse).
* Validate incoming receipts (ensure they parse).
* Extract `txOutPublicKey` from incoming receipts.